rinex/src/macros.rs

```diff
@@ -0,0 +1,61 @@
+//! Macros and helpers
+
+/// Creates an [crate::Sv] from given string description,
```

Comment:
Maybe

```suggestion
/// Creates an [`crate::Sv`] from given string description,
```

(and the same below)